### ISSUE TITLE
The results are not good

### ISSUE THREAD

**Issue by gildorgood** [timsainb/Tensorflow-MultiGPU-VAE-GAN#7]
Hi,

I download your network and train it but the results are not good.
My results are so blurry and not human-like.

How many epochs are needed to converge?
I observed that  the results collapse after 70~80 epochs.
All hyper-parameters are same as your version.

Is there any optimization tip?
Any advice would be very helpful.

Thank you.

**Comment by timsainb** [comment]
Hello gildorgood,
When I ran it, it would take the night to converge to something good on a titanx, not sure how many epochs. You may need to play around with hyperparameters a bit? To optimize it, I was keeping track of how the generator and discriminator loss were competing. From what I've heard from others, things have changed a bit for new Tensorflow versions. Currently with networks that I'm running, I do not use prettytensor at all and instead use Tensorflow built in functions from TF 1.0+:
https://github.com/timsainb/Tensorflow-MultiGPU-VAE-GAN/issues/4#issuecomment-302512016
The same goes for optimization.

**Comment by gildorgood** [comment]
I appreciate your reply.  I still have some issues that I cannot resolve. 
If it is not offending, can you please share your pre-trained network?
